### PR TITLE
adding support for enableGPUValidationMode

### PR DESCRIPTION
**Pull request by LouisLWang**

### CODE REVIEW

File: Sources/ProjectSpec/Scheme.swift [hunk 1]
```diff
@@ -416,6 +419,11 @@ extension Scheme.Run: JSONObjectConvertible {
         } else {
             enableGPUFrameCaptureMode = XCScheme.LaunchAction.defaultGPUFrameCaptureMode
         }
+        if let gpuValidationMode: String = jsonDictionary.json(atKeyPath: "enableGPUValidationMode") {
+            enableGPUValidationMode = XCScheme.LaunchAction.GPUValidationMode.fromJSONValue(gpuValidationMode)
+        } else {
+            enableGPUValidationMode = XCScheme.LaunchAction.defaultGPUValidationMode
+        }
```
Comment (yonaskolb) [inline Sources/ProjectSpec/Scheme.swift:426]: You'll also need to encode in `Scheme.Run.toJSONValue`

**Comment by yonaskolb** [comment]
The tests are failing as the updated fixtures have not been committed. Just rerun the tests and commit the diffs.
Could you also please add a changelog entry

**Comment by LouisLWang** [comment]
> The tests are failing as the updated fixtures have not been committed. Just rerun the tests and commit the diffs. Could you also please add a changelog entry

The Tests have finished. Please review this PR when you available.

**Comment by yonaskolb** [comment]
Just missed one comment above
> Could you also please add a changelog entry

And also while I notice, just a doc update at https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#common-build-action-options

**Comment by louiswang002** [comment]
> Just missed one comment above
> 
> > Could you also please add a changelog entry
> 
> And also while I notice, just a doc update at https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#common-build-action-options

Done.

**Comment by LouisLWang** [comment]
> Just missed one comment above
> 
> > Could you also please add a changelog entry
> 
> And also while I notice, just a doc update at https://github.com/yonaskolb/XcodeGen/blob/master/Docs/ProjectSpec.md#common-build-action-options

Done.

**Review by yonaskolb — APPROVED** [review]
Thanks @LouisLWang. I will add the changelog entry on master for you

**Pull request merged**